frontend/src/components/SingleModuleCard.tsx

```diff
@@ -99,12 +94,6 @@ const SingleModuleCard: React.FC<SingleModuleCardProps> = ({
 
           {dropdownOpen && (
             <div className="absolute right-0 z-20 mt-2 w-40 rounded-md border border-gray-200 bg-white shadow-lg dark:border-gray-700 dark:bg-gray-800">
-              <button
```

Comment:
Hmm... We needed to hide the whole thing, there's no point to keep the three dots with broken dropdown (this is clicked into/active state):
<img width="409" height="151" alt="Screenshot 2025-09-09 at 7 58 40 PM" src="https://github.com/user-attachments/assets/d329b0ad-1078-4806-9245-e06ccf6f1ced" />